lib/ruby_smb/dcerpc/request.rb

```diff
@@ -3,66 +3,113 @@ module Dcerpc
     # The Request PDU as defined in
     # [The request PDU](http://pubs.opengroup.org/onlinepubs/9629399/chap12.htm#tagcjh_17_06_04_09)
     class Request < BinData::Record
+      PTYPE = PTypes::REQUEST
+
       endian :little
 
-      pdu_header :pdu_header, label: 'PDU header'
+      # PDU Header
+      pdu_header :pdu_header, label: 'PDU header common fields'
```

Comment:
Both packets have the same `common fields`, I've updated the response `label` to reflect that. This set of fields is common to every PDU and it is completed with PDU-specific fields to form the complete PDU header (as defined in [RPC PDU Encodings](https://pubs.opengroup.org/onlinepubs/9629399/chap12.htm)).

I realized that the `label` logic has been borrowed from SMB packets, which all inherit from `RubySMB::GenericPacket`. This class implement some logic that uses the `label` parameter. Since DCERPC packets do not inherit from this class, `label` parameter is not used. That said, I did not remove them since I think it gives a better description of each field and we might want to use them in the future.